src/lib/stitching/vortex/schema.ts

```diff
@@ -34,7 +34,11 @@ export const executableVortexSchema = ({
         ]
       : []),
     new RenameTypes((name) => {
-      return `Analytics${name}`
+      if (name === "PriceInsights" || name === "MarketPriceInsights") {
```

Comment:
These values were not prefixed with `Analytics` when they were coming from Diffusion.

Reply:
2. The message `Field 'MarketPriceInsights.updatedAt' changed type from 'ISO8601DateTime' to 'AnalyticsDateTime'` suggests we should do something similar with DateTime....but it doesn't seem to be as simple as the `BigInt` case. I'm not sure why `ISO8601` was being prefixed when it came from diffusion, but to avoid issues with clients using these types, I think we want that type name not to change.

Reply:
```suggestion
      if (["PriceInsights", "MarketPriceInsights", "BigInt"].includes(name)) {
```

3. (non-blocking) With more than two items in that comparison list, it might also be more readable as checking for the presence in an array.

Reply:
4. These messages also suggest there are other types we need to exclude in the rename, but I'd want to take a closer look to be sure: 

```
Type 'PriceInsightSort' was removed
Type 'PriceInsightConnection' was removed
Type 'PriceInsightEdge' was removed
Field 'marketPriceInsights' was removed from object type 'Query'
Field 'priceInsights' was removed from object type 'Query'
```